### PR TITLE
PTX design rework + fixes

### PR DESCRIPTION
**Pull request by FrederikBolding**
- Reworked PTX design implementation
- Fixed a bunch of CSS/styling issues

### CODE REVIEW

File: src/components/PoweredByText.tsx [hunk 1]
```diff
@@ -47,11 +47,7 @@ const providers: Record<string, PoweredByProvider> = {
   NANSEN: {
     icon: 'nansenLogo',
     text: translateRaw('POWERED_BY'),
-    fontSize: '12px',
-    css: `
-      position: absolute;
-      bottom: ${SPACING.BASE};
-    `
+    fontSize: '12px'
```
Comment (blurpesec) [inline src/components/PoweredByText.tsx:50]: Use `SPACING`!

**Comment by LuckyLuker** [comment]
[ch6381]

File: src/components/PoweredByText.tsx [hunk 2]
```diff
@@ -47,11 +47,7 @@ const providers: Record<string, PoweredByProvider> = {
   NANSEN: {
     icon: 'nansenLogo',
     text: translateRaw('POWERED_BY'),
-    fontSize: '12px',
-    css: `
-      position: absolute;
-      bottom: ${SPACING.BASE};
-    `
+    fontSize: `${FONT_SIZE.XS}`
```
Comment (vilsbole) [inline src/components/PoweredByText.tsx:50]: 👍

File: src/components/GeneralStepper/GeneralStepper.tsx [hunk 3]
```diff
@@ -13,6 +13,7 @@ export interface StepperProps {
   defaultBackPathLabel?: string;
   completeBtnText?: string; // Text for btn to navigate out of flow in last step.
   wrapperClassName?: string;
+  basic?: boolean;
```
Comment (vilsbole) [inline src/components/GeneralStepper/GeneralStepper.tsx:16]: The prop name is ambiguous. eg. What is a non-basic `GeneralStepper`?
Can we choose a more declarative name or comment please

**Pull request merged**